checkov/common/runners/object_runner.py

```diff
@@ -203,7 +203,7 @@ def add_python_check_results(
                         bc_check_id=check.bc_id,
                         check_name=check.name,
                         check_result=result,
-                        code_block=self.definitions_raw[file_path][start - 1:end + 1],
+                        code_block=self.definitions_raw[file_path][start - 1:end + 2],
```

Comment:
why `end+2`?
do we know for sure that always `len(self.definitions_raw[file_path]) >=  end+2` ?